client/modules/IDE/reducers/project.js

```diff
@@ -1,7 +1,10 @@
 import * as ActionTypes from '../../../constants';
+const generate = require('project-name-generator');
+
+const generatedName = generate({ words: 2 }).dashed;
 
 const initialState = {
-  name: 'Hello p5.js'
+  name: `${generatedName}`
```

Comment:
Oh, right! The way to make this work is to make initialState into a function rather than a variable. See the files reducer for an example of how to do this.